Geometry/TrackerNumberingBuilder/interface/GeometricDet.h

```diff
@@ -3,7 +3,7 @@
 
 #include "CondFormats/GeometryObjects/interface/PGeometricDet.h"
 #include "DetectorDescription/Core/interface/DDExpandedView.h"
```

Comment:
This include should be removed in this PR or a later PR.  The `using GeoHistory = std::vector<DDExpandedNode>;` in this file and several others in MTD and `TrackerNumberingBuilder` needs to be removed or migrated. Could you and @fabiocos please coordinate on this task?